10326.xml

```diff
@@ -0,0 +1,724 @@
+        <Type>Integer</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[HARI S-Map Requests number by the robot.]]></Description>
+      </Item>	  
+      <Item ID="1107">
+        <Name>Successful HARI S-Voice Requests</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Successful HARI S-Voice Requests number by the robot.]]></Description>
+      </Item>
+      <Item ID="1108">
+        <Name>Successful HARI S-Vision Requests</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Successful HARI S-Vision Requests number by the robot.]]></Description>
+      </Item>
+      <Item ID="1109">
+        <Name>Successful HARI S-Motion Requests</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Successful HARI S-Motion Requests number by the robot.]]></Description>
+      </Item>
+      <Item ID="1110">
+        <Name>Successful HARI S-Map Requests</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[Successful HARI S-Map Requests number by the robot.]]></Description>
+      </Item>	  
+      <Item ID="1111">
+        <Name>Questions Answered</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[How many questions have been answered by the robot.]]></Description>
+      </Item>
+      <Item ID="1112">
+        <Name>Unknown Questions</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration></RangeEnumeration>
+        <Units/>
+        <Description><![CDATA[How many questions can't be answered by the robot.]]></Description>
+      </Item>
+	  <Item ID="1113">
+        <Name>Mileage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Optional</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>M</Units>
```

Comment:
Incorrect Unit submitted - The Units defined in OMA LwM2M Objects are defined by SenML - Sensor Measurement Lists.

Reply:
M（meter）, the unit of distance.

Reply:
``M`` should be replaced with ``m``, as per SenML

Reply:
Changed(M to m, M/H to m/h), thanks.